docs/concepts/pipelines/parallel.md

```diff
@@ -0,0 +1,107 @@
+# Parallel Pipelines
+Nodes in a pipeline can run in parallel, allowing multiple operations to proceed simultaneously.
```

Comment:
I know you make it clear later that if you have a setup like this, that the output node will see the output from its previous nodes twice (once for each), but I am wondering if we shouldn't mention it here as well and / or point to the explanation later in the doc?